drv/cosmo-seq-server/src/main.rs

```diff
@@ -69,7 +69,15 @@ enum Trace {
         seq_state: Result<fmc_periph::A0Sm, u8>,
     },
     PowerDownError(drv_cpu_seq_api::SeqError),
-
+    Coretype {
+        coretype0: bool,
+        coretype1: bool,
+        coretype2: bool,
+        sp5r1: bool,
+        sp5r2: bool,
+        sp5r3: bool,
+        sp5r4: bool,
+    },
```

Comment:
i briefly wondered whether there was a more space-efficient way to represent this, but it's only 7 bytes, and isn't the longest entry type in this ringbuf, so it doesn't actually matter. carry on!